shuup/core/models/_stored_basket.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+# This file is part of Shuup.
```

Comment:
This stored basket should be at the API branch already. Shouldn't be multivendor related.